src/stream-net/stream-net.csproj

```diff
@@ -3,7 +3,7 @@
     <DefineConstants>TRACE;DEBUG;NETCORE</DefineConstants>
   </PropertyGroup>
   <PropertyGroup>
-    <TargetFrameworks>net45;net461;netcoreapp1.0;netstandard1.6;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net45;net461;netcoreapp1.0;netcoreapp2.2;netstandard1.6;netstandard2.0</TargetFrameworks>

```

Comment:
Added dot net core 2.2 as targeted framework

Reply:
Should add netcore3.1 too

Reply:
sure will add, possibly 5.0 too

Reply:
That sounds great :+1: 

We can bump the major and kill old stuff then.